src/SigmaDSP.cpp

```diff
@@ -180,6 +180,24 @@ void SigmaDSP::volume(uint16_t startMemoryAddress, float dB)
   safeload_write(startMemoryAddress, volume);
 }
 
+/**
+ * @brief control mute/unmute block without slew
+ *
+ * @param startMemoryAddress DSP memory address
+ * @param flag true or false
+ */
+void SigmaDSP::mute(uint16_t startMemoryAddress, bool flag)
+{
```

Comment:
Please use the same indentation style as elsewhere in the library. Two white spaces, not four.